brainsets_pipelines/peterson_brunton_pose_trajectory_2022/pipeline.py

```
@@ -0,0 +1,748 @@
+parser.add_argument("--redownload", action="store_true")
+parser.add_argument("--reprocess", action="store_true")
+parser.add_argument(
+    "--resample_rate",
+    type=float,
+    default=500.0,
+    help="ECoG resample rate in Hz",
+)
+parser.add_argument(
+    "--filter_ecog",
+    action="store_true",
+    help="Apply 70–150 Hz bandpass and Hilbert envelope",
```

Ruff (0.15.10)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)